autopush/settings.py

```diff
@@ -204,6 +205,13 @@ def create_initial_message_tables(self):
         """
         last_month = get_rotating_message_table(self._message_prefix, -1)
         this_month = get_rotating_message_table(self._message_prefix)
+        # Verify the table exists
+        try:
+            this_month.describe()
+        except Exception:
+            # Create the message table
+            create_rotating_message_table(prefix=self._message_prefix)
+            this_month = get_rotating_message_table(self._message_prefix)
         self.message_tables = {
             last_month.table_name: Message(last_month, self.metrics),
```

Comment:
That's fine because it means this isn't a real production deploy. The current month is needed for a new start, ongoing deployments will run a set of scripts that hasn't landed yet to ensure tables are around.